public/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<header>
+	<link rel="stylesheet" type="text/css" href="style.css">
+	<h1>Reservation Component</h1>
+</header>
+<body>		
+	<div id="app">
+	<script src="../dist/bundle.js"></script>
```

Comment:
Don't think your script tag for your bundle.js should be inside your App's rendering "div" element. Should be the last element in "body" element.